nni/contrib/compression/quantization/lsq_quantizer.py

```diff
@@ -58,10 +63,21 @@ def __init__(self, model: torch.nn.Module, config_list: List[Dict], evaluator: E
         self.evaluator: Evaluator
         self.is_init = False
 
+        self.check_validation()
         self.register_scale()
         self.register_lsq_apply_method()
         self.register_track_func()
 
+    def check_validation(self):
+        for _, ts in self._target_spaces.items():
```

Comment:
`for ts in self._target_spaces.values():`

Reply:
done